ffi/diplomat/Cargo.toml

```diff
@@ -147,14 +144,11 @@ serde = { version = "1.0", default-features = false, optional = true }
 diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "8d125999893fedfdf30595e97334c21ec4b18da9" }
 diplomat-runtime = { git = "https://github.com/rust-diplomat/diplomat", rev = "8d125999893fedfdf30595e97334c21ec4b18da9" }
 
+log = { version = "0.4", optional = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-# Logging is automagical in wasm, we only need this for native
+# Logging uses diplomat_runtime bindings in wasm, we only need this for native
 simple_logger = { version = "4.1.0", optional = true }
-log = { version = "0.4", optional = true }
-
-[target.'cfg(target_arch = "wasm32")'.dependencies]
-log = "0.4"
```

Comment:
Issue: Is it correct to say that with this change, logging gets disabled in wasm mode by default? That would be a regression.

Reply:
No, but gn uses `default-features = false`